grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -88,6 +88,10 @@ class ApiController {
         def localizationMode
         def locale = localizationService.getCurrentLocale()
         Object[] emptyArgs = [] as Object []
+        def translationModeLocale = grailsApplication.config.openboxes.locale.translationModeLocale
+        // If current locale is equal to translation mode locale, we are in localization mode
+        session.useDebugLocale = locale == new Locale(translationModeLocale)
```

Comment:
Instead of sprinkling this logic everywhere, we should probably have it done in a filter. InitializationFilters seems to be the best candidate but I'm open to suggestions.

Reply:
sounds good to me, I haven't thought about such a solution